tsl/test/expected/compression_ddl.out

```diff
@@ -1613,25 +1613,28 @@ EXPLAIN (costs off) SELECT device_id, count(*)
 FROM compression_insert
 GROUP BY device_id
 ORDER BY device_id;
-                                                               QUERY PLAN                                                                
------------------------------------------------------------------------------------------------------------------------------------------
+                                                                     QUERY PLAN                                                                      
+-----------------------------------------------------------------------------------------------------------------------------------------------------
  Finalize GroupAggregate
-   Group Key: _hyper_31_110_chunk.device_id
-   ->  Merge Append
-         Sort Key: _hyper_31_110_chunk.device_id
-         ->  Custom Scan (VectorAgg)
-               ->  Custom Scan (DecompressChunk) on _hyper_31_110_chunk
-                     ->  Index Scan using compress_hyper_32_111_chunk_device_id__ts_meta_min_1__ts_me_idx on compress_hyper_32_111_chunk
-         ->  Custom Scan (VectorAgg)
-               ->  Custom Scan (DecompressChunk) on _hyper_31_112_chunk
-                     ->  Index Scan using compress_hyper_32_113_chunk_device_id__ts_meta_min_1__ts_me_idx on compress_hyper_32_113_chunk
-         ->  Custom Scan (VectorAgg)
-               ->  Custom Scan (DecompressChunk) on _hyper_31_114_chunk
-                     ->  Index Scan using compress_hyper_32_115_chunk_device_id__ts_meta_min_1__ts_me_idx on compress_hyper_32_115_chunk
-         ->  Partial GroupAggregate
-               Group Key: _hyper_31_114_chunk.device_id
-               ->  Index Only Scan using _hyper_31_114_chunk_compression_insert_device_id_time_idx on _hyper_31_114_chunk
-(16 rows)
+   Group Key: _hyper_31_114_chunk.device_id
+   ->  Sort
+         Sort Key: _hyper_31_114_chunk.device_id
+         ->  Gather
+               Workers Planned: 2
+               ->  Parallel Append
```

Comment:
This should be GatherMerge above Sort, will be addressed here: https://github.com/timescale/timescaledb/pull/7547